### ISSUE TITLE
Raidboss: questions about migrating Another Aloalo trigger .ts file to savage

### ISSUE THREAD

**Issue by cordy5134** [quisquous/cactbot#5937]
Hi,
I'm new to the cactbot developing and recently was trying to find out how to create Another Aloalo Savage triggers before the official one comes out by cloning the another_alo branch. Here is what I did:
1. Make a copy of another_aloalo_island.ts and renamed to another_aloalo_island_savage.ts
2. changed the id and zoneId to "AnotherAloaloIslandSavage "in the triggerSet, but keeps the timeline file the same.
3. got a act log file for savage to find skill ID.
Here's the problem:
When I load the trigger into act, I got the warning of : 
warning overridden by "06-ew/dungeon/another_aloalo_island_savage.ts" 

Is there anything I need to do other than finding the skill IDs for savage and replace them in the file? Or should I make a new id for each trigger in the savage file?
Thanks for your help!

**Comment by quisquous** [comment]
I updated the PR with changes to `util/sync_files.ts` to generate these files and some initial stub files. If you update the ids in that script and then run it, it will update the savage files. Or you can edit it manually too I guess.

I'm still doing this content blind so please don't share ids or any other feedback (sorry). Hope that helps. I'll maybe get into savage next week, but scheduling is hard.

**Comment by cordy5134** [comment]
Thank you for response.
I was able to test the triggers in the raid emulator and it seems fine to my eyes. However I cannot use it directly in the game and I got this log from overlay plugin. It seems that I need to change something in order to make it work outside the emulator right?

> `[2023/11/20 16:37:38] Info: OverlayControl: [webpack-dev-server] [object Event] (file:///C:/Users/70902/AppData/Roaming/Advanced%20Combat%20Tracker/Plugins/cactbot-another_alo_laovan/cactbot-another_alo/dist/ui/config/config.bundle.js)

> [2023/11/20 16:37:38] Info: OverlayControl: [webpack-dev-server] Trying to reconnect... (file:///C:/Users/70902/AppData/Roaming/Advanced%20Combat%20Tracker/Plugins/cactbot-another_alo_laovan/cactbot-another_alo/dist/ui/config/config.bundle.js)`

**Comment by quisquous** [comment]
Are you running the dev server and switching your raidboss overlay to use that, e.g. `http://localhost:8080/ui/raidboss/raidboss.html`

See: https://github.com/quisquous/cactbot/blob/main/CONTRIBUTING.md#validating-changes-via-webpack

**Comment by cordy5134** [comment]
I was trying the triggers in the raid emulator, haven't successfully import to ACT yet. I was able to successfully load web log data and show triggers in the emulator. I should try the method you mentioned when I got chance.

P.S.: About the proxy-agent thing, I just figured out it was broke when I fix the vulnerabilities after I call "npm install". It needs to be a --force fix so that has to be why. Maybe this is a dependency version issue but it's not related to this one anyway
